docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -99,18 +105,23 @@ Constructors
 
 .. cppkokkos:function:: RangePolicy()
 
-   Default Constructor uninitialized policy.
+   Construct an uninitialized policy.
 
-.. cppkokkos:function:: template<class ... InitArgs> RangePolicy(const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
+.. cppkokkos:function:: RangePolicy(index_type begin, index_type end)
 
-   Provide a start and end index as well as optional arguments to control certain behavior (see below).
+   Construct a policy with a start index and an end index.
 
-.. cppkokkos:function:: template<class ... InitArgs> RangePolicy(const ExecutionSpace& space, const int64_t& begin, const int64_t& end, const InitArgs ... init_args)
+.. cppkokkos:function:: RangePolicy(index_type begin, index_type end, ChunkSize cs)
 
-   Provide a start and end index and an ``ExecutionSpace`` instance to use as the execution resource, as well as optional arguments to control certain behavior (see below).
+   Construct a policy with a start index, an end index and a ChunkSize (see below).
 
-Optional ``InitArgs``:
-^^^^^^^^^^^^^^^^^^^^^^
+.. cppkokkos:function:: RangePolicy(const execution_space& space, index_type begin, index_type end)
```

Comment:
Please move this `InitArgs` information to the "Parameters" section, and consolidate coherently with all of the other guidance on parameter arguments.